code/game/objects/structures/xeno.dm

```diff
@@ -854,6 +870,7 @@ TUNNEL
 	var/recharge_rate = 10
 	///Countdown to the next time we generate a jelly
 	var/nextjelly = 0
+	var/mob/living/carbon/xenomorph/creator
 
```

Comment:
Harddel protection missing

Reply:
What do you mean? It's nulled out on Destroy